src/emulator/app/src/discord.cpp

```diff
@@ -0,0 +1,44 @@
+#ifdef USE_DISCORD_RICH_PRESENCE
+#include <app/discord.h>
+#include <discord_register.h>
+#include <discord_rpc.h>
+
+// Credits to the RPCS3 Project
+
+namespace discord {
+void initialize(const std::string &application_id) {
```

Comment:
```suggestion
void init(const std::string &application_id) {
```
to fit the rest of our modules